lib/srv/proxy.go

```diff
@@ -247,7 +247,10 @@ func (t *proxySubsys) proxyToHost(
 	// if port is 0, it means the client wants us to figure out
 	// which port to use
 	useExactPort := len(t.port) > 0 && t.port != "0"
-	ips, _ := net.LookupHost(t.host)
+	log.Debugf("proxy connecting to host=%v port=%v, exact port=%v\n", t.host, t.port, useExactPort)
+
+	ips, err := net.LookupHost(t.host)
+	log.Debugf("host lookup (%v)=%v, err=%v", t.host, ips, err)
```

Comment:
the problem with `err=` is that it will be 99% of the time equal to `err=nil` and grep catches it all the time, so we should try to make it less verbose even for debug.